test/unit/org/apache/cassandra/cql3/validation/miscellaneous/CrcCheckChanceTest.java

```diff
@@ -36,31 +36,12 @@
 import org.apache.cassandra.io.util.RandomAccessReader;
 import org.apache.cassandra.utils.FBUtilities;
 
-
 public class CrcCheckChanceTest extends CQLTester
 {
-
-
-    @Test
-    public void testChangingCrcCheckChanceNewFormat() throws Throwable
-    {
-        testChangingCrcCheckChance(true);
-    }
-
     @Test
-    public void testChangingCrcCheckChanceOldFormat() throws Throwable
-    {
-        testChangingCrcCheckChance(false);
-    }
-
-
-    public void testChangingCrcCheckChance(boolean newFormat) throws Throwable
+    public void testChangingCrcCheckChance()
     {
-        //Start with crc_check_chance of 99%
-        if (newFormat)
-            createTable("CREATE TABLE %s (p text, c text, v text, s text static, PRIMARY KEY (p, c)) WITH compression = {'class': 'LZ4Compressor'} AND crc_check_chance = 0.99;");
-        else
-            createTable("CREATE TABLE %s (p text, c text, v text, s text static, PRIMARY KEY (p, c)) WITH compression = {'class': 'LZ4Compressor', 'crc_check_chance' : 0.99}");
+        createTable("CREATE TABLE %s (p text, c text, v text, s text static, PRIMARY KEY (p, c)) WITH compression = {'class': 'LZ4Compressor'} AND crc_check_chance = 0.99;");
```

Comment:
What will happen if the user still set the crc_check_chance through compression ?